### PR TITLE
edit dynamic task

### PR DESCRIPTION
**Pull request by MatusKovacKASV**
Hello. I have a problem on receiving data from Golang (get task). it's throwing me CORS Policy error.. Can you tell me if  the problem is in my code or not ? :) 

specific lines :  [events].tsx : 19 - 36

### CODE REVIEW

File: front-end/pages/events/[events].tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+interface challengeData{
+	title: string,
+	description: string
+}
+
+const HandleGetTask = async (router: any) => {
+
+	try{
+		const url = "http://10.2.130.98:9090/get-language-tasks";
```
Comment (anton-cornak) [inline front-end/pages/events/[events].tsx:22]: add this to environment variables

File: front-end/pages/events/[events].tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+import LargeHeading from "@/components/ui/LargeHeading";
+import TextAnimation from "@/components/ui/TextAnimation";
+import axios from "axios";
+import type { Metadata } from "next";
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+
+export const metadata: Metadata = {
+	title: "VISMA WARS",
+	description: "KASV project for Vsima employees",
```
Comment (anton-cornak) [inline front-end/pages/events/[events].tsx:11]: Vsima 😄

**Comment by anton-cornak** [comment]
> Hello. I have a problem on receiving data from Golang (get task). it's throwing me CORS Policy error.. Can you tell me if the problem is in my code or not ? :)
> 
> specific lines : [events].tsx : 19 - 36

ask @EmanuelHuzvar and try to solve it together. It will work locally if you run go server and front-end on localhost, since there is no CORS violation.

**Comment by MatusKovacKASV** [comment]
> > Hello. I have a problem on receiving data from Golang (get task). it's throwing me CORS Policy error.. Can you tell me if the problem is in my code or not ? :)
> > specific lines : [events].tsx : 19 - 36
> 
> ask @EmanuelHuzvar and try to solve it together. It will work locally if you run go server and front-end on localhost, since there is no CORS violation.

We tried it already but still same issue idk why...

File: front-end/pages/events/[events].tsx (hunk 1 above)
Comment (MatusKovacKASV) [reply front-end/pages/events/[events].tsx:22]: Okay I did it. We will try with Emanuel again and let you know if it works thanks :)

**Comment by MatusKovacKASV** [comment]
> Are you sure you merged your branch with main? Because all of sudden, you have a lot of files you want to push to main, which doesn't seem right, including images.

**Comment by anton-cornak** [comment]
Ok, try everything out locally using `yarn dev`, then try to build the app using `yarn build` and if there are some errors, fix them please. Otherwise it's good to go.

**Comment by anton-cornak** [comment]
fixed build

**Review by anton-cornak — APPROVED** [review]

**Pull request merged**